.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: |
+            This issue is marked as stale due to requiring feedback for 30 days. Please respond within 5 days or this issue will be closed due to inactivity.
```

Comment:
```suggestion
            This issue is marked as stale due because author feedback has been requested for 30 days with no response. Please respond within 5 days or this issue will be closed due to inactivity.
```